### DISCUSSION TITLE
multiple Select with checkmarks render behavior

### DISCUSSION THREAD

**Discussion by Daniel-Raz-Supersmart** [lookfirst/mui-rff#619] [Q&A]
**Describe the bug**
I'm trying to figure hot work properly with Select multiple checkmarks

**To Reproduce**
1. Go to  `https://stackblitz.com/edit/react-jtv7me-nrde93?file=demo.js`
2. Click on Select
3. Every option is checked

**Expected behavior**
First, empty and check by pressed

**Desktop (please complete the following information):**
 - OS: Windos 10
 - Browser chrome

**Additional context**
What is the right way to do this?

**Comment by lookfirst** [comment]
I've migrated this to a discussion.

I tried to run your stackblitz and nothing at all renders. 

![2021-12-13 22-29-10](https://user-images.githubusercontent.com/85355/145868009-67f888d1-542b-4b18-8341-85b5960c1cb0.png)

What I suggest you do is simplify your code significantly and come up with a super simple test case that demonstrates the problem clearly. My guess is that if you do that, you'll find out what the issue is in your code. Right now, it is a bit of a mess.

There are also examples of using select boxes in the example project within mui-rff. I can see that the codesandbox is currently broken for some reason, but those examples should still run just fine.

I hope that gets you moving towards a solution to your issue.

**Reply by Daniel-Raz-Supersmart** [reply]
thanks for the quick response, I updated the stackblitz appreciate it if you can look it again
